lib/rubycritic/generators/html/assets/javascripts/application.js

```diff
@@ -49,7 +49,7 @@ $("#churn-vs-complexity-graph-container").highcharts({
   },
   plotOptions: {
     series: {
-      turboThreshold: 1500
+      turboThreshold: 15000
```

Comment:
I'd rather set this to `0` so this won't ever be a problem.

I'm also thinking on how we could make this configurable :thinking: 
Maybe add a control on the page to set this value and re-render the chart. But not sure if it'd be that useful. 

@AzizHarrington what do you think?